apps/api/src/search/search.service.ts

```diff
@@ -1,16 +1,18 @@
 import { Injectable } from '@nestjs/common';
 import { Repository } from 'typeorm';
 import { InjectRepository } from '@nestjs/typeorm';
-import { User as UserEntity } from 'types';
+import { User as UserEntity, Userfront } from 'types';
+import { TransformUserService } from 'src/TransformUser/TransformUser.service';
 
 @Injectable()
 export class SearchService {
   constructor(
     @InjectRepository(UserEntity)
     private readonly usersRepository: Repository<UserEntity>,
+    private readonly transformUserService: TransformUserService,
   ) {}
 
-  async findAll(letters: string): Promise<UserEntity[]> {
+  async findAll(letters: string): Promise<Userfront[]> {
```

Comment:
I have updated this method to perform a more efficient search

Reply:
i don't understand how i loose your work... i think it's when i rebase it. i don't know how to get it again. you steal have it ?

Reply:
done